src/components/AvatarCropModal/ImageCropView.js

```diff
@@ -46,7 +46,7 @@ const propTypes = {
 const defaultProps = {
     imageUri: '',
     containerSize: 0,
-    panGestureEventHandler: () => {},
+    panGesture: Gesture.Pan(),
```

Comment:
Is it safe to use the same instance of a gesture as a default?

Reply:
That depends on what you mean by `safe` as it shouldn't cause crashes. One thing that could happen is the same gesture instance being attached to different views, which would result in an undefined behavior. Since the default value here has no config nor callbacks set, it shouldn't have any negative effects (and I didn't observe any during testing).

Reply:
That's exactly what I meant by this queston, thanks!